assets/css/stripeStyle.css

```diff
@@ -132,3 +111,172 @@ input:focus {
 .hidden {
 	display: none;
 }
+
+
+
+form {
+	display: flex;
+	flex-wrap: wrap;
+	padding: 0.5rem;
+}
+
+form h2 {
+	width: 100%;
+	padding: 0.5rem;
+	color: #304FFE;
```

Comment:
Codacy found an issue: [Expected "#304FFE" to be "#304ffe" (color-hex-case)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5886534)